### ISSUE TITLE
Some Properties are undefined in textLayerRendered

### ISSUE THREAD

**Issue by Julian-B90** [stephanrauh/ngx-extended-pdf-viewer#2494]
```html
<ngx-extended-pdf-viewer
    filter="span"
    id="transaction-pdf-preview"
    [src]="src"
    [ignoreKeys]="['j', 'k', 'F4', 'CTRL++', 'CTRL+-', 'CTRL+S', 'CTRL+F', 'CTRL+G', 'P', 'N', 'HOME', 'END', 'S', 'H', 'R', 'Shift+R', 'Ctrl+Alt+P']"
    [showBorders]="false"
    [showToolbar]="true"
    (handToolChange)="onHandToolEnabledChange($event)"
    [zoom]="'page-width'"
    [enableDragAndDrop]="false"
    [language]="'de'"
    [textLayer]="true"
    [showHandToolButton]="true"
    (pageRendered)="drawElements($event)"
    (textLayerRendered)="onTextLayerRendered($event)"
    [height]="'100%'"
    [customToolbar]="additionalButtons"
    data-testid="pdf-viewer"
/>
```

```ts
onTextLayerRendered(data: TextLayerRenderedEvent) {
        const pageNumber = data.pageNumber;
        const numTextDivs = data.numTextDivs;
        const numTextDivs_2 = data.source.textLayer?.numTextDivs;

        console.log({
            pageNumber,
            numTextDivs,
            numTextDivs_2
        });
    }
```

```
{pageNumber: 1, numTextDivs: undefined, numTextDivs_2: undefined}
```

Installed Version:
    "ngx-extended-pdf-viewer": "^21.1.2",

**Comment by stephanrauh** [comment]
You're right. I'm a confused. Maybe the internal API of pdf.js has changed? Or maybe it was my mistake from the start? Be that as it may, consider it done!

**Comment by stephanrauh** [comment]
Your bug fix has landed with version 21.3.1.
